blake2/Cargo.toml

```diff
@@ -23,7 +23,7 @@ hex-literal = "0.4"
 default = ["std"]
 std = ["digest/std"]
 reset = [] # Enable reset functionality
-simd = []
-simd_opt = ["simd"]
-simd_asm = ["simd_opt"]
+#simd = []
+#simd_opt = ["simd"]
+#simd_asm = ["simd_opt"]
```

Comment:
We probably should finish #228 before publishing blake2 v0.11.

Reply:
Yeah, definitely been meaning to do it. Would be great to get it over the finish line.